app/src/main/java/com/concordia/mcga/fragments/NavigationFragment.java

```diff
@@ -206,15 +210,50 @@ public void onMapReady(GoogleMap googleMap) {
         applyCustomGoogleMapsStyle();
         Campus.populateCampusesWithBuildings();
         addBuildingMarkersAndPolygons();
-
+        currentCampus=closestCampus(distanceBetween(((MainActivity) getActivity()).getGpsManager().getLocation(),Campus.SGW.getMapCoordinates()),distanceBetween(((MainActivity) getActivity()).getGpsManager().getLocation(),Campus.LOY.getMapCoordinates()));
         updateCampus();
     }
 
+    /**
+     *
+     * @param point1: First LatLng to compare
+     * @param point2: Second LatLng to compare
+     * @return a double calculated by the different in LatLng by SphericalUtil
+     * This method calculates the distance between two LatLng variables.
+     * To be used with method "closestCampus".
+     */
+    public Double distanceBetween(LatLng point1, LatLng point2){
+        if(point1==null||point2 == null){
+            return null;
+        }
+        else{
+            return SphericalUtil.computeDistanceBetween(point1, point2);
+        }
+    }
+
+    /**
+     *
+     * @param distance1: Distance returned from myLocation to SGW Campus
+     * @param distance2: Distance returned from myLocation to LOY Campus
+     * @return: Returns the campus it's closest to
+     * This method is used to return a campus depending on the user's current location by comparing the distance between the user and both campus's
+     * location using the previous "distanceBetween" method
+     */
+    public Campus closestCampus(double distance1, double distance2){
```

Comment:
I get what these are from the comments, but you may want to rename them to like say
"distance1" -> myLocationToSGW
"distance2" -> myLocationToLOY